compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1418,21 +1418,21 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
     /// Given a function definition like:
     ///
     /// ```rust
+    /// use std::fmt::Debug;
     /// fn test<'a, T: Debug>(x: &'a T) -> impl Debug + 'a {
     ///     x
     /// }
     /// ```
     ///
     /// we will create a TAIT definition in the HIR like
-    ///
-    /// ```
-    /// type TestReturn<'a, T, 'x> = impl Debug + 'x
-    /// ```
-    ///
-    /// and return a type like `TestReturn<'static, T, 'a>`, so that the function looks like:
+    /// type TestReturn<'a, 'x, T> = impl Debug + 'x;
```

Comment:
Instead of deleting the codeblock, you can use ```rust,ignore``` at the header to prevent rustdoc from compiling or running it. You could also use `text` as the header to avoid treating it as a test at all, although you would lose syntax highlighting.

Reply:
My apologies for being late on this PR; I didn't have the time.